_posts/2022-08-15-requiring-mfa-on-popular-gems.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Requiring MFA on popular gem maintainers
+layout: post
+author: Jenny Shen
+author_email: jenny.shen@shopify.com
+---
+<p align="center">
+    <img src="/images/gem-with-thumbs-up-mfa-dropshadow.png" alt="Doodle of a RubyGem wearing a MFA hat, giving a thumbs up" width="300"/>
+</p>
+
+Two months ago, we outlined our [commitment](https://blog.rubygems.org/2022/06/13/making-packages-more-secure.html) to making Ruby’s supply chain more secure. To combat account takeovers — the second most common software supply chain attack —  we announced a policy to require MFA on at least the top-100 RubyGems packages.
```

Comment:
We should spell out "MFA" and define the acronym before using the acronym elsewhere. 

> Two months ago, we outlined our commitment to making Ruby’s supply chain more secure. To combat account takeovers — the second most common software supply chain attack — we announced a policy to require 👉 multi-factor authentication (MFA) 👈 on at least the top-100 RubyGems packages.

Reply:
Yes agreed, great catch! 🚀